src/utils.ts

```diff
@@ -77,19 +76,45 @@ const safeTerminalNavItem = (data: any): TerminalNavItem => ({
   idHTML: safeString(data?.idHTML),
 });
 
-const safePlan = (data: any): Plan => ({
-  planType: safeString(data?.planType),
-  description: safeString(data?.description),
-  itemDescription: safeString(data?.itemDescription),
-  planName: safeString(data?.planName),
-  isSubscribers: safeBoolean(data?.isSubscribers),
-  maxSubscribers: safeNumber(data?.maxSubscribers),
-  remainingCredits: safeNumber(data?.remainingCredits),
-  buttonText: safeString(data?.buttonText),
-  buttonUrl: safeString(data?.buttonUrl),
-  pendingFreeUpgrade: safeBoolean(data?.pendingFreeUpgrade),
-  isMonthlyByEmail: safeBoolean(data?.isMonthlyByEmail),
-});
+const safePlan = (data: any): Plan => {
+  /** Doppler plan types
+   * 1: 'free',
+   * 2: 'monthly-deliveries',
+   * 3: 'prepaid',
+   * 4: 'subscribers',
+   * 5: 'agencies',
+   * 6: 'agencies',
+   * 7: 'free',
+   * 8: 'agencies',
+   */
+  const planTypes = [
+    undefined,
+    "free",
+    "monthly-deliveries",
+    "prepaid",
+    "subscribers",
+    "agencies",
+    "agencies",
+    "free",
+    "agencies",
+  ];
```

Comment:
Creo que si en lugar de definirlo cómo un array lo definís como un objeto el comentario no sería necesario.

```suggestion
  const planTypes = {
    0: undefined,
    1: "free",
    2: "monthly-deliveries",
    3: "prepaid",
    4: "subscribers",
    5: "agencies",
    6: "agencies",
    7: "free",
    8: "agencies",
  };
```

Por otro lado, creo que no sería necesario especifdicar el `0`

Reply:
👍🏻